test/tests/kernels/adv_diff_reaction/adv_diff_reaction_test.i

```diff
@@ -113,6 +113,12 @@
 []
 
 [Outputs]
+  [png]
+    type = PNGObject
+    PNGFile = "PNGTEST"
+    resolution = 50
+    inColor = false
```

Comment:
Not this test either but feel free to copy this test into the outputs directory to create a new test (see comment below)